e2e/e2e-script.sh

```diff
@@ -141,6 +138,13 @@ addJsonToFile "mcRGName" $MC_RESOURCE_GROUP_NAME
 addJsonToFile "clusterID" $CLUSTER_ID
 addJsonToFile "subID" $SUBSCRIPTION_ID
 
+set +x
+jq -r 'keys[] as $k | "export \($k)=\(.[$k])"' fields.json
```

Comment:
@alexeldeib this line is troublesome. need to fine another way to export the var to be able to set +x around it.